scripts/generate-style-code.js

```diff
@@ -6,12 +6,14 @@ const spec = require('mapbox-gl-style-spec').latest;
 var parseCSSColor = require('csscolorparser').parseCSSColor;
 
 global.camelize = function (str) {
+  str = str === undefined ? 'undefined' : str;
```

Comment:
What prompted this change, and the corresponding one in `camelizeWithLeadingLowercase`?